package.json

```diff
@@ -12,12 +12,13 @@
     "db:up": "docker-compose up -d",
     "db:migrate": "postgrator",
     "db:down": "docker-compose down",
-    "lint": "eslint .",
-    "test": "tap"
+    "lint": "eslint --ext .",
```

Comment:
are you sure this does what you think it does? don't you need to specify extensions?